src/DurableTask.AzureStorage/Storage/Queue.cs

```diff
@@ -48,7 +48,7 @@ public async Task AddMessageAsync(QueueMessage queueMessage, TimeSpan? visibilit
             await this.azureStorageClient.MakeStorageRequest(
                 (context, cancellationToken) => this.cloudQueue.AddMessageAsync(
                     queueMessage.CloudQueueMessage,
-                    null /* timeToLive */,
+                    TimeSpan.FromSeconds(-1) /* timeToLive */,
```

Comment:
Can you update the comment to explain why we're setting this to `TimeSpan.FromSeconds(-1)`?

Something like:
```suggestion
                    TimeSpan.FromSeconds(-1) /* disable TTL */,
```